python/cuspatial/cuspatial/core/gis.py

```diff
@@ -142,51 +161,55 @@ def point_in_polygon(
     closed polygons: the first and last coordinate of each polygon must be
     the same.
 
-    params
-    test_points_x: x-coordinate of test points
-    test_points_y: y-coordinate of test points
-    poly_offsets: beginning index of the first ring in each polygon
-    poly_ring_offsets: beginning index of the first point in each ring
-    poly_points_x: x closed-coordinate of polygon points
-    poly_points_y: y closed-coordinate of polygon points
-
     Parameters
     ----------
-    {params}
+    test_points_x
+        x-coordinate of test points
+    test_points_y
+        y-coordinate of test points
+    poly_offsets
+        beginning index of the first ring in each polygon
+    poly_ring_offsets
+        beginning index of the first point in each ring
+    poly_points_x
+        x closed-coordinate of polygon points
+    poly_points_y
+        y closed-coordinate of polygon points
 
     Examples
     --------
-        result = cuspatial.point_in_polygon(
-            cudf.Series([0, -8, 6.0]]), # x coordinates of 3 query points
-            cudf.Series([0, -8, 6.0]), # y coordinates of 3 query points
-            cudf.Series([1, 2], index=['nyc', 'dc']), # ring positions of
-                    # two polygons each with one ring
-            cudf.Series([4, 8]), # positions of last vertex in each ring
-            # polygon coordinates, x and y. Note [-10, -10] and [0, 0] repeat
-            # the start/end coordinate of the two polygons.
-            cudf.Series([-10, 5, 5, -10, -10, 0, 10, 10, 0, 0]),
-            cudf.Series([-10, -10, 5, 5, -10, 0, 0, 10, 10, 0]),
-        )
-        # The result of point_in_polygon is a DataFrame of Boolean
-        # values indicating whether each point (rows) falls within
-        # each polygon (columns).
-        print(result)
-                    nyc            dc
-        0          True          True
-        1          True         False
-        2         False          True
-
-        # Point 0: (0, 0) falls in both polygons
-        # Point 1: (-8, -8) falls in the first polygon
-        # Point 2: (6.0, 6.0) falls in the second polygon
+
+    Test whether 3 points fall within either of two polygons
+
+    >>> result = cuspatial.point_in_polygon(
+        [0, -8, 6.0],                             # test_points_x
+        [0, -8, 6.0],                             # test_points_y
+        cudf.Series([0, 1], index=['nyc', 'dc']), # poly_offsets
```

Comment:
Maybe use hudson river in place of dc, since it partially overlaps with nyc (may be less confusing).
```suggestion
        [0, -8, 6.0],                             # test_points_y
        cudf.Series([0, 1], index=['nyc', 'hudson river']), # poly_offsets
```